sigproc/src/OmnibusSigProc.cxx

```diff
@@ -1656,7 +1657,7 @@ bool OmnibusSigProc::operator()(const input_pointer& in, output_pointer& out)
         }
     }
 
-    auto sframe = new Aux::SimpleFrame(in->ident(), in->time(), ITrace::shared_vector(itraces), in->tick(), m_cmm);
+    auto sframe = new Aux::SimpleFrame(in->ident(), in->time(), ITrace::shared_vector(itraces), in->tick(), in->masks());
```

Comment:
don't change to och.channel for OmnibusSigProc output cmm?

Reply:
I think we didn't want to change to och.channel to keep the global channel mapping